test/unit/util.js

```diff
@@ -2,14 +2,8 @@ const test = require('ava')
 
 const util = require('../../dist/server/util')
 
-test('get score static', t => {
-  const score = util.scores.getScore('static', 100, 100, 20)
-
-  t.is(score, 100)
-})
-
 test('get score dynamic', t => {
-  const score = util.scores.getScore('dynamic', 100, 500, 20)
-
-  t.is(score, 484)
+  t.is(util.scores.getScore(100, 500, 0), 500)
```

Comment:
Do we want to hardcode test values? Or just make sure that the returned value seems reasonable, ie returned score is positive. Otherwise, our tests essentially dictate a hardcoded formula.